Examples/LocationManager/CommonTests/CommonTests.swift

```diff
@@ -27,18 +27,18 @@ class LocationManagerTests: XCTestCase {
     let locationManagerSubject = PassthroughSubject<LocationManager.Action, Never>()
 
     store.environment.locationManager.authorizationStatus = { .notDetermined }
-    store.environment.locationManager.create = { _ in locationManagerSubject.eraseToEffect() }
+    store.environment.locationManager.delegate = { locationManagerSubject.eraseToEffect() }
```

Comment:
I renamed this to `delegate` since it's really just the lifecycle around delegate actions.